chaosblade-exec-plugin/chaosblade-exec-plugin-jvm/src/test/java/com/alibaba/chaosblade/exec/plugin/jvm/script/java/JavaUtilsTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 1999-2019 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.chaosblade.exec.plugin.jvm.script.java;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class JavaUtilsTest {
+
+    @Test
+    public void testGetClassName() {
```

Comment:
Add assert that checks for an actual class name rather than ""?

Reply:
Done